src/containers/Login/index.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 
+import Page from 'containers/Login/page';
```

Comment:
1. Why using `containers` over `screens` here if there are no Redux implementation
2. This is not following [Compass conventions](https://compass.nimblehq.co/development/code-conventions/react/#containers) to name containers

```
import Login from 'containers/Login.jsx';

## for Screens

import Login from 'screens/Login/index.jsx';

```

Reply:
Changed in [e4c0e56](https://github.com/rafayet-monon/surveyor/pull/12/commits/e4c0e561d462b1c7317a86867f3ac321092e5f47)